aries-site/src/examples/templates/forms/RequiredFieldsExample.js

```diff
@@ -90,6 +92,8 @@ export const RequiredFieldsExample = () => {
           >
             <Select options={superPower} id="superPower" name="superPower" />
           </FormField>
+          {/* TO DO - enhance formfield-htmlfor-id to support Select */}
```

Comment:
Do we have an issue to track this?

Reply:
Nevermind, I just found it https://github.com/grommet/eslint-plugin-grommet/issues/46

Reply:
Good catch -- let me adjust the comment

Reply:
Done.